ingenialink/dictionary.py

```diff
@@ -128,21 +138,6 @@ class Dictionary(ABC):
     DICT_ENUMERATIONS = "./Enumerations"
     DICT_ENUMERATIONS_ENUMERATION = f"{DICT_ENUMERATIONS}/Enum"
 
-    class AttrRegDict:
```

Comment:
Why did you remove this? The idea is avoid use literal strings

Reply:
This was used to save the keys of the dictionary used to fill the register attributes before creating a Register instance. As we are not doing so anymore, this enum is not used.

Reply:
But now you always use literals for the keys used for "register.attrib", if some day dictionary keys change we must look for all the literals and replace them, It's better to use constants, enums, or something like that.

Reply:
Yes, but this class was not used for that purpose. Would you like to use it to store the xdf keys?

Reply:
Ok, I see, it is out of the issue scope, it's ok.